packages/babel-plugin-proposal-class-properties/test/fixtures/nested-class/super-property-in-key/output.js

```diff
@@ -27,7 +27,7 @@ function (_Hello) {
     babelHelpers.classCallCheck(this, Outer);
     _this = babelHelpers.possibleConstructorReturn(this, babelHelpers.getPrototypeOf(Outer).call(this));
 
-    var _babelHelpers$get$cal = babelHelpers.get(babelHelpers.getPrototypeOf(Outer.prototype), "toString", babelHelpers.assertThisInitialized(_this)).call(_this);
```

Comment:
Didn't really look into it, but is it supposed to call `assertThisInitialized` multiple twice like this?

Reply:
No, but it won't hurt. Like I said in the OP, this output is worse because I'm not optimizing any special cases yet, I just need to refactor code out of here so I can work on access-to-function-calls so we can get Private props. Then I can start work on refactoring all of the classes transform.